cli/src/main/kotlin/commands/ReporterCommand.kt

```diff
@@ -88,16 +92,29 @@ class ReporterCommand : CliktCommand(
         .file(mustExist = false, canBeFile = false, canBeDir = true, mustBeWritable = false, mustBeReadable = false)
         .required()
 
-    private val reporters by option(
+    private val reportFormats by option(
         "--report-formats", "-f",
         help = "The list of report formats that will be generated."
-    ).convert { reporterName ->
-        // Map upper-cased reporter names to their instances.
-        val reporters = Reporter.ALL.associateBy { it.reporterName.toUpperCase() }
-        reporters[reporterName.toUpperCase()]
-            ?: throw BadParameterValue("Reporters must be contained in ${reporters.keys}.")
+    ).convert { format ->
+        reporters[format.toUpperCase()]
+            ?: throw BadParameterValue("Report formats must be one or more of ${reporters.keys}.")
     }.split(",").required()
 
+    private val reportOptions by option(
+        "--report-option", "-O",
+        help = "Specify a report-format-specific option to be passed to the reporter implementation. The key is the" +
+                "(case-insenstive) name of the report format, and the value is an arbitrary key-value pair. For " +
+                "example: -O webapp=css=my-style.css"
```

Comment:
Is this a feature you plan to implement? If not, is there another feature you plan to implement that you could use as an example?

Reply:
I'm planning to use this new feature with an upcoming reporter that's not implemented yet. As an example for that reporter does not make sense at this time, I chose to use a "fictional" example for an existing reporter, which I plan to change once the new reporter is there. Would you prefer to have no example at this point?

Reply:
+1 on dropping the example until we have a reporter using this new feature. Prefer not to give false expectations as users have previously asked me for an option to pass their own CSS file to the WebApp.

Reply:
Will make that change, and tracking the CSS feature as a new issue: https://github.com/oss-review-toolkit/ort/issues/2643